sdk/src/rand.rs

```diff
@@ -9,28 +9,16 @@ use crate::{sys, SyscallResult};
 /// The supplied output buffer must have at least 32 bytes of capacity.
 /// If this syscall succeeds, exactly 32 bytes will be written starting at the
 /// supplied offset.
-pub fn get_chain_randomness(
-    dst: i64,
-    round: ChainEpoch,
-    entropy: &[u8],
-) -> SyscallResult<[u8; RANDOMNESS_LENGTH]> {
-    let ret = unsafe {
-        sys::rand::get_chain_randomness(dst, round, entropy.as_ptr(), entropy.len() as u32)?
-    };
+pub fn get_chain_randomness(round: ChainEpoch) -> SyscallResult<[u8; RANDOMNESS_LENGTH]> {
+    let ret = unsafe { sys::rand::get_chain_randomness(round)? };
     Ok(ret)
 }
 
 /// Gets 32 bytes of randomness from the beacon system (currently Drand).
 /// The supplied output buffer must have at least 32 bytes of capacity.
 /// If this syscall succeeds, exactly 32 bytes will be written starting at the
 /// supplied offset.
-pub fn get_beacon_randomness(
-    dst: i64,
-    round: ChainEpoch,
-    entropy: &[u8],
-) -> SyscallResult<[u8; RANDOMNESS_LENGTH]> {
-    let ret = unsafe {
-        sys::rand::get_beacon_randomness(dst, round, entropy.as_ptr(), entropy.len() as u32)?
-    };
+pub fn get_beacon_randomness(round: ChainEpoch) -> SyscallResult<[u8; RANDOMNESS_LENGTH]> {
+    let ret = unsafe { sys::rand::get_beacon_randomness(round)? };
     Ok(ret)
```

Comment:
nit: no need for the temporary `ret` variable, the entire function can be written as:

```
unsafe { sys::rand::get_beacon_randomness(round) }
```